### PR TITLE
Portfolio mobile menu  toggle navigation menu

### PR DESCRIPTION
**Pull request by Bria222**
- added toggle navigation menu
- added javascript linters
- Added CSS to style hamburger icon
- added javascript to toggle navigation elements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,36 +4,44 @@
...
 
-            <div class="menu-top">
-              <ul class="menu-item">
-                <li class="menu-links">Portfolio</li>
-                <li class="menu-links">About</li>
-                <li class="menu-links">Contact</li>
-              </ul>
+    <div class="wrapper">
+      <!-- mobile navigation section -->
+      <div class="mobile-nav">
```
Comment (ShoiraTa) [inline index.html:19]: - [x] The hamburger button and menu should follow the design from Figma, but I noticed that on mobile screens, the logo gets next to hamburger button, please make sure that logo and hamburger button are alight on the left and right side as shown in Figma:

Your design  | Figma design
------------- | -------------
Screenshot 2022-04-06 at 13 51 12 | Screenshot 2022-04-06 at 13 51 47

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi @Bria222,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost


 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bria222** [comment]
@ShoiraTa i have implemented the changes thanks kindly check

**Review by Kingstalux — APPROVED** [review]
Hi @Bria222,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6d43496a436773336e575157664a5a7650412f67697068792d646f776e73697a65642d6c617267652e676966


Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**